src/main/java/com/configcat/FetchResponse.java

```diff
@@ -33,10 +33,13 @@ public Entry entry() {
     }
 
     public String error() {
```

Comment:
It seems to me that this class just stores some data temporarily so we can propagate it to methods higher up in the call chain, doesn't it? If so, we shouldn't "materialize" the message here.

```suggestion
    public Object error() {
```